exercises/random_number.cpp

```diff
@@ -4,3 +4,17 @@
   Output:
   The random number is: 4
 */
+
+#include <iostream>
+#include<cstdlib>
+#include<ctime>
+
+using namespace std;
+
+int main() {
+  srand(time(NULL));
+
+  cout << "The random number is: " << (rand() % 10);
+
+  return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```